examples/knowledgebasebot/actions.py

```diff
@@ -0,0 +1,16 @@
+from rasa_sdk.knowledge_base.storage import InMemoryKnowledgeBase
+from rasa_sdk.knowledge_base.actions import ActionQueryKnowledgeBase
+
+
+class ActionMyKB(ActionQueryKnowledgeBase):
+    def __init__(self):
+        # load knowledge base with data from the given file
+        knowledge_base = InMemoryKnowledgeBase.load("knowledge_base_data.json")
+
+        # overwrite the representation function of the hotel object
+        # by default the representation function is just the name of the object
+        knowledge_base.set_representation_function_of_object(
```

Comment:
is `object` the right word? maybe rather`type` or something from the knowledge base domain - e.g. how does grakn call them?

Reply:
grakn calls them entities, but entities is confusing as we already use the term entities for our NER. Alternative terms would be nodes, but nodes is only related to graph databases and not used in relation databases. So I went with object. Let me know if you have a better idea.